src/common/meta/src/kv_backend/postgres.rs

```diff
@@ -0,0 +1,619 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::any::Any;
+use std::borrow::Cow;
+use std::sync::Arc;
+
+use snafu::ResultExt;
+use tokio_postgres::types::ToSql;
+use tokio_postgres::{Client, NoTls};
+
+use super::{KvBackend, TxnService};
+use crate::error::{ConnectPostgresSnafu, Error, PostgresFailedSnafu, Result, StrFromUtf8Snafu};
+use crate::kv_backend::txn::{Txn as KvTxn, TxnResponse as KvTxnResponse};
+use crate::kv_backend::KvBackendRef;
+use crate::rpc::store::{
+    BatchDeleteRequest, BatchDeleteResponse, BatchGetRequest, BatchGetResponse, BatchPutRequest,
+    BatchPutResponse, CompareAndPutRequest, CompareAndPutResponse, DeleteRangeRequest,
+    DeleteRangeResponse, PutRequest, PutResponse, RangeRequest, RangeResponse,
+};
+use crate::rpc::KeyValue;
+
+/// Posgres backend store for metasrv
+pub struct PgStore {
+    // TODO: Consider using sqlx crate.
+    client: Client,
+}
+
+const EMPTY: &[u8] = &[0];
+
+const METADKV_CREATION: &str =
```

Comment:
Let's change the default table name to `greptime_metakv`. It would be better to allow the user to configure it.

Reply:
done and left a todo to allow users to configure the meta table name